src/Components/Card.tsx

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
+import Image from 'next/image';
 
-type CardProps = {
-    
+interface CardProps {
+  imageUrl: string;
+  title: string;
+  price: string;
+  rating: string;
+}
+
+const Card = ({ imageUrl, title, price, rating }: CardProps) => {
```

Comment:
It would be better if we rename our components to have more strict names related to each major Component. The name card was just initially setup as a generic test name. Perhaps some name like DressCard, DesignPageCard etc.

Reply:
renamed it DesignCard